src/components/POLine/Vendor/FieldRefNumberType.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'redux-form';
+import { Select } from '@folio/stripes/components';
+import { RequiredRefNumber } from '../../Utils/Validate';
+
+const REF_NUMBER_TYPE = {
+  suppliersContinuation: "Supplier's continuation order",
+  suppliersUnique: "Supplier's unique order line reference number",
+  agentsUnique: "Agent's unique subscription reference number",
+  librarysContinuation: "Library's continuation order number",
+  internalNumber: 'Internal vendor number',
+};
+
+class FieldRefNumberType extends Component {
```

Comment:
could you please try to use functional component here? it doesn't look like it's going to have an internal state here